impl-aggregator/wrapper/aggregator/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+pub mod wrap;
+pub use wrap::*;
+
+pub fn get_animals(args: ArgsGetAnimals) -> Vec<String> {
```

Comment:
I belive the aggregator example should be better than this. It should actually call one or more of implementations depending on conditions.

I have an idea will do some changes in this